src/main/java/TextView.java

```diff
@@ -30,10 +45,17 @@ public void update(Observable o, Object arg)
 	 * abbreviation for a given piece and the outer frame of the board.
 	 * 
 	 * @param board Current board state.
-	 * @throws InvalidPositionException
 	 */
 	void drawBoard(Board board)
 	{
+		int rowCount = 1;
+		System.out.println("     a   b   c   d   e   f   g   h    ");
+		System.out.print(SPACE);
+		
+		for (int i =0; i < board.GRID_SIZE; i++){
+			System.out.print("----");
+		}
+		System.out.println("  ");
```

Comment:
- [ ] Is this space supposed to be a different length than `SPACE`?. If so, part of me wants to have a variable declared for this different sized space as well. Call it something like `SMALL_SPACE` or something.